dali/test/dali_test_single_op.h

```diff
@@ -182,6 +182,18 @@ class DALISingleOpTest : public DALITest {
     SetBatchSize(32);
   }
   inline void TearDown() override {
+    for (auto &jpeg : jpeg_decoded_) {
+      delete [] jpeg;
```

Comment:
same here, I'd like to see this memory managed by a vector instead if possible

Reply:
Done